apps/oozie/src/oozie/views/dashboard.py

```diff
@@ -968,7 +968,13 @@ def submit_external_job(request, application_path):
     params_form = ParametersFormSet(request.POST)
 
     if params_form.is_valid():
-      mapping = dict([(param['name'], param['value']) for param in params_form.cleaned_data])
+      if application_name == 'coordinator.xml':
+        external_properties = Submission(request.user, fs=request.fs, jt=request.jt).get_external_parameters(application_path)
+        non_parameters = ParameterForm.get_non_parameters(external_properties)
+        mapping = non_parameters
+        mapping.update(dict([(param['name'], param['value']) for param in params_form.cleaned_data]))
+      else :
+        mapping = dict([(param['name'], param['value']) for param in params_form.cleaned_data])
```

Comment:
Then outside of the if we do a single?

        mapping.update(dict([(param['name'], param['value']) for param in params_form.cleaned_data]))

Reply:
Yes, done my last commit [7386ff7](https://github.com/cloudera/hue/pull/1128/commits/7386ff7bddc49b80e97ad51adf630644dec9451f)